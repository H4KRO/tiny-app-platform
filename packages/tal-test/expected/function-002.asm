main()
  entry:
    FunctionRef       name: func_0
func_0([object Object], [object Object])
  entry:
    Local             name: a
    Local             name: b
    Intrinsic         operation: INTRINSIC_ADD
