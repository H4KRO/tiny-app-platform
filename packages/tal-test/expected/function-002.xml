<program>
  <function>
    <binaryoperator>
      <local />
      <local />
    </binaryoperator>
  </function>
</program>
